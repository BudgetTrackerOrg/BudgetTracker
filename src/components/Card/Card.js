import React from 'react'
import { View, Platform } from 'react-native'

export default props => {
    return (
        <View
            style={{
                ...props.style,
                backgroundColor: '#fff',
                borderRadius: 10,
                ...Platform.select({
                    ios: {
                        shadowColor: 'rgba(0,0,0, .2)',
                        shadowOffset: { height: 0, width: 0 },
                        shadowOpacity: 1,
                        shadowRadius: 1
                    },
                    android: {
                        elevation: 2
                    }
                })
            }}
        >
            {props.children}
        </View>
    )
}
