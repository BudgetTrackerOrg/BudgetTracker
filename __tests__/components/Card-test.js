import 'react-native'
import React from 'react'
import Card from '../../src/components/Card/Card'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
    const tree = renderer.create(<Card />).toJSON()
    expect(tree).toMatchSnapshot()
})
