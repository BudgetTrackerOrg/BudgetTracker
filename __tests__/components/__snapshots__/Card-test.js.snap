// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderRadius": 10,
      "shadowColor": "rgba(0,0,0, .2)",
      "shadowOffset": Object {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 1,
    }
  }
/>
`;
